client/http/http_sender_test.go

```diff
@@ -15,19 +15,22 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package client
+package http
```

Comment:
Sounds good, will get them updated. client/rpc and client/http were such pretty names, but those conflicts are hard on readability.